src/System.Private.Xml.Linq/tests/misc/LoadSaveAsyncTests.cs

```diff
@@ -16,6 +16,7 @@ namespace CoreXml.Test.XLinq
 {
     public class LoadSaveAsyncTests : BridgeHelpers
     {
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]   //Async methods defined only on Core
```

Comment:
These tests are included in in the test csproj when `'$(TargetGroup)'==''` and the tests compile for net463 because I had added an APICompatBaseline for it in `System.Xml.XDocument` when adding these new APIs. Is that the correct way?